### DISCUSSION TITLE
3.2.5版本的标绘和测量的点或线不显示

### DISCUSSION THREAD

**Discussion by webdq** [zouyaoji/vue-cesium#504] [General]
包版本
cesium : 1.113.0
vue-cesium : 3.2.5

![325](https://github.com/zouyaoji/vue-cesium/assets/17735764/0f49db88-fd79-4879-8b45-51edcba71aa3)

并且有如下2个警告

> VcCollectionPoint Add event linstener of primitiveRemoved failed, try to upgrade Cesium to latest version.

> VcCollectionLabel Add event linstener of primitiveAdded failed, try to upgrade Cesium to latest version.

切换到 3.2.4 版本，标绘和测量可以显示

![324](https://github.com/zouyaoji/vue-cesium/assets/17735764/481b1e6d-be83-4db5-8031-fb5cc2d85a6b)

**Comment by zouyaoji** [comment]
我测试了没重现哈。根据你控制台信息，怀疑你的Cesium版本不是1.113.0的。

**Reply by zouyaoji** [reply]
重现了，在很高的视角下确实有问题。我看看。
